application/backend/src/main/resources/apps/typerefinery/components/widgets/ticker/script.js

```diff
@@ -100,7 +101,13 @@ $(document).ready(function (e) {
 
     // Data can be updated via TMS Connection.
     if (componentTopic && componentHost) {
-      component.setAttribute("id", componentTopic);
+      // component.setAttribute("id", componentTopic);
```

Comment:
Why did you comment on this line?

If you are commenting on this, LIne 36 won't work, did you try this code?

Please check them again.